### PR TITLE
Migrate unique field in Book table

### PR DESCRIPTION
**Pull request by ghostbear**
This adds a migration to remove the unique field in the Book table. It may not be possible just to remove it with an alter table but it is possible to remove it by shuffling tables around.

### CODE REVIEW

**Comment by alessandrojean** [comment]
Have you managed to get it working with Flyway? When I tried this approach it was throwing some exceptions because of the Pragma statement.

**Comment by ghostbear** [comment]
Yeah, it migrated with Flyway. Had some issues with JooQ but that was due to the book foreign key in the other tables.
image

**Comment by alessandrojean** [comment]
I will try on my database backup and will merge in a moment 👍🏻

File: server/src/main/resources/application.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ logging:
 
 tankobon:
   database:
-    file: \${tankobon.config-dir}/app_database.sqlite
+    file: \${tankobon.config-dir}/database.sqlite
```
Comment (alessandrojean) [inline server/src/main/resources/application.yml:17]: Can you undo the change in the `application-localdb.yml` as well, please? It's the only thing missing.

**Comment by alessandrojean** [comment]
Tested and confirmed to be working 👍🏻

**Review by alessandrojean — APPROVED** [review]
